README.md

```diff
@@ -291,7 +290,13 @@ Description of arguments for these scripts are as follows. See examples below.
 1. cdr20180206.sql uploaded to all-of-us-workbench-private-cloudsql
 1. public20180206.sql uploaded to all-of-us-workbench-public-cloudsql
 
-#### TODO import these dumps to cloudsql databases  
+#### Import a dump to cloudsql instance.
+`./project.rb cloudsql-import  --account all-of-us-workbench-test@appspot.gserviceaccount.com --project all-of-us-workbench-test --instance workbenchtest --sql-dump-file cdr20180206.sql --bucket all-of-us-workbench-private-cloudsql`
+##### Wait an hour or so for above to finish before running another import on same instance
```

Comment:
It's kind of a drag that it takes this long. We should probably script importing both of these so you don't have to wait an hour before kicking off the second command?

Or at any rate, tell folks to run a bash command that does both...

Reply:
Options are:
1)  to make one big sql dump. It will be like 6 gig. 
2) Or have different instances for public.
3) Or just live with it.

Given we just do this 4 times a year I opt for 3 now and improve later.
For example, we could run it to test, staging, and dev and prod all at once if they all have separate cloudsql instances. 

 I think this is good general script and we can use it in some other script to get a better workflow as we start using it.

Reply:
Or 4. have a command run here that does both sequentially... what I'm suggesting.

Waiting around for an hour to run another command that runs for an hour is a recipe for frustration. :)

Reply:
Note this command could just be:

./project.rb ....private  && ./project.rb .... public

Reply:
We can't do that unless the databases are in different instances. That is the problem. import doesn't like it. 
Can you run ./project.rb inside of a running ./project.rb script ?

Reply:
foo && bar runs foo followed by bar if foo succeeds. Isn't that what we want?

Reply:
As discussed offline: this runs asynchronously. Please updated doc to indicate that and how to know when it's done.

Reply:
Updated.